regress/core/Makefile.in

```diff
@@ -137,43 +140,13 @@ TESTS = \
 	swapordinates \
 	summary \
 	temporal \
+	temporal_knn \
 	tickets \
 	twkb \
 	typmod \
 	wkb \
 	wkt \
-	wmsservers
-
-ifeq ($(shell expr $(POSTGIS_PGSQL_VERSION) "<"  95),1)
-	# Index supported KNN only available in PostgreSQL 9.1 and higher
-	# For 9.5 and higher, use the recheck suite instead
-	TESTS += knn
```

Comment:
Since knn won't be used anymore, can we delete the file too?

Reply:
:+1: